pkg/graph/schema/types/mto_common_solution_contractor_translation.graphql

```diff
@@ -0,0 +1,8 @@
+"""
+Represents a translation of a contractor for a common solution
+"""
+type MTOCommonSolutionContractorTranslation {
+  key: TranslationField! @goTag(key: "db", value:"mto_common_solution_key")
+  contractorTitle: TranslationField @goTag(key: "db", value:"contractor_title")
```

Comment:
I think this should be mandatory (a translation is always required, even if the value is not required)

Reply:
ok cool making TranslationField non nullable